docs/guides/devops-lifecycle/integrate-web-modeler-in-ci-cd.md

```diff
@@ -16,6 +16,10 @@ import TabItem from "@theme/TabItem";
 
 [Web Modeler](../../components/modeler/about-modeler.md) serves as a powerful tool for the development and deployment of processes and process applications. While Web Modeler simplifies one-click deployment for development, professional teams often rely on continuous integration and continuous deployment (CI/CD) pipelines for automated production deployments. The [Web Modeler API](/apis-tools/web-modeler-api/index.md) facilitates integration of Web Modeler into these pipelines, aligning with team practices and organizational process governance.
 
+- For low-risk processes, you can use the Web Modeler [process application development lifecycle](/docs/components/modeler/web-modeler/process-application-pipeline.md) to quickly develop and progress low-risk process application releases through the stages of a typical development lifecycle. [Milestone comparison](/docs/components/modeler/web-modeler/milestones.md#compare-milestones) (Visual and XML diffing) and [Git Sync](/docs/components/modeler/web-modeler/git-sync.md) provide a powerful combination.
```

Comment:
I raised the question on line 22 because git sync felt out of place. I was left wondering: "powerful for what"? I proposed a use case that it helps with below
```suggestion
For low-risk processes, you can use the Web Modeler [process application development pipeline](/docs/components/modeler/web-modeler/process-application-pipeline.md) to quickly develop and progress process application releases through the stages of a standard development life cycle. [Milestone comparison](/docs/components/modeler/web-modeler/milestones.md#compare-milestones) (Visual and XML diffing) and [Git Sync](/docs/components/modeler/web-modeler/git-sync.md) provide a powerful combination for collaborating between team members using both Web and Desktop Modelers.
```

FYI Git sync has 3 use cases:
* provide a detailed history of changes and backup (as files deleted from Web Modeler cannot be recovered)- relevant to all use cases
* integrate desktop and web modelers - relevant to all use cases
* integrate with a CI/CD pipeline - mutually exclusive with the out-of-the-box deployment stages and promotion flow

Knowing this information, would you change anything else?

Reply:
The use cases info could be useful on the Git Sync page itself, but not sure if any of it fits here in the process applications docs? Could you provide more context please?